src/ui/CategoryFacet/CategoryFacet.ts

```diff
@@ -235,22 +346,82 @@ export class CategoryFacet extends Component {
     return this.categoryValueRoot.children;
   }
 
+  private renderValues(categoryFacetResult: ICategoryFacetResult, numberOfRequestedValues: number) {
+    this.show();
+    let sortedParentValues = this.sortParentValues(categoryFacetResult.parentValues);
+    let currentParentValue: CategoryValueParent = this.categoryValueRoot;
+    let needToTruncate = false;
+    let pathOfLastTruncatedParentValue: string[];
+
+    const numberOfItemsInFirstSlice = Math.floor(CategoryFacet.NUMBER_OF_VALUES_TO_KEEP_AFTER_TRUNCATING / 2);
```

Comment:
Could an error happen if `NUMBER_OF_VALUES_TO_KEEP_AFTER_TRUNCATING` is odd, and thus cannot split evenly in two parts ?

Reply:
You're right, `numberOfItemsInSecondSlice` should use `Math.ceil`. Good catch